modules/ROOT/pages/getting-started.adoc

```diff
@@ -20,9 +20,11 @@ The SDK provides a Maven plugin and a Maven archetype to create new projects. Th
 
 Install and verify that the following tools are working in your local environment:
 
-* Java Development Kit 17 (JDK 17) or earlier to compile and build your custom Java code.
-** For preexisting connectors, xref:java-version-support.adoc#jdk_8[JDK 8 might be needed].
-* Apache Maven 3.9.8 or later to manage your project’s build.
+* Java Development Kit 8 (JDK 8) to compile and build your custom Java code.
++
+NOTE: The maximum supported version for running your applications is Java 17 for Mule runtime 4.6.x and later. During the packaging of your app, all code, including third-party dependencies, *must* be compiled for Java 8.
```

Comment:
I believe we do not translate content that uses the bold syntax: *

```suggestion
NOTE: The maximum supported version for running your applications is Java 17 for Mule runtime 4.6.x and later. During the packaging of your app, all code, including third-party dependencies, must be compiled for Java 8.
```

You could make that sentence more active in voice and perhaps emphasize that we're talking about compilation